src/components/App/index.jsx

```diff
@@ -18,6 +20,14 @@ import store from '../../data/store';
 
 const AppWrapper = () => {
   const apiClient = getAuthenticatedHttpClient();
+
+  if (process.env.HOTJAR_APP_ID) {
+    const { initHotjar } = useHotjar();
+    useEffect(() => {
```

Comment:
not being fully knowledgable on `useEffect`, does this just allow us to async init hot jar as a hook?

Reply:
That sounds....plausible. 😅 
Legit just took the snippet from the react-use-hotjar readme.